src/app/scenario/scenario.component.html

```diff
@@ -29,10 +29,28 @@ <h3>Scenarios</h3>
 </div>
 <div class="clr-row">
   <div class="clr-col-4">
-    <clr-datagrid [clrDgSingleSelected]="unusedSelectedScenario" (clrDgSingleSelectedChange)="editScenario($event)" [clrDisableSelection]="!selectRbac">
+
+    <clr-dropdown>
+      <button class="dropdown-toggle" clrDropdownTrigger>
+        <span *ngIf="category==''">All</span>
+        <span *ngIf="category=='__none__'">No Category</span>
+        <span *ngIf="category !='' && category != '__none__' ">{{category}}</span>
+        <clr-icon shape="caret down"></clr-icon>
+      </button>
+      <clr-dropdown-menu clrPosition="top-left" *clrIfOpen>
+        <div class="dropdown-item" (click)="setCategoryFilter('')">All</div>
```

Comment:
add 'clrDropdownItem' to close dropown after selection. Add to no Category Filter aswell